src/test/java/org/jboss/xavier/integrations/migrationanalytics/business/VMWorkloadInventoryCalculatorTest.java

```diff
@@ -113,6 +116,7 @@ public void calculate_jsonV1_0_0_Given_ShouldReturnCalculatedValues() throws IOE
         expectedModel.setVersion("6.7.2");
         expectedModel.setProduct("VMware vCenter");
         expectedModel.setScanRunDate(new SimpleDateFormat("yyyy-M-dd'T'hh:mm:ss.S").parse("2019-09-18T14:52:45.871Z"));
+        expectedModel.setHasCpuHotRemove(false);
```

Comment:
You are only setting expectancy for 1 of the 3 new filds... any reason for that ?

Reply:
Each of the fields is set to trigger the flag in a different VM in the test payload, and the expectedValue test only looks at one of these VMs.

Reply:
I've changed it so they are all set in that VM